features/config/osgi/del/src/main/java/org/opennms/features/config/osgi/del/MigratedServices.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2021 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 2021-2021 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.features.config.osgi.del;
+
+import java.util.Set;
+
+public class MigratedServices {
+
+    final public static Set<String> PIDS =  Set.of(
```

Comment:
Is this the best place to list the PIDs? (Would be nice if it was dynamic from the CM service.)

Reply:
The problem is the following:
We are in the delegator here.
The delegator has at its startup time no access to cm.
it uses this information to determine if it needs to delegate to cm or file storage

What we could potentially do is a two fold approach:
1. check if the pid starts with "org.opennms": Assumption is that all our osgi configurations start with _org.opennms_.
2. then ask cm if that pid is available

Reply:
Gotcha. Yeah, this can work for now but we'll ultimately want to revisit.